### ISSUE TITLE
a11y-light has 7 foreground colors whereas a11y-high-contrast-light has 8

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/accessible-pygments#35]
Title says it all. After #33, `a11y-high-contrast-light` has a shade of magenta (`#912583`) but `a11y-light` does not have a shade of magenta.

Do we want to resolve this discrepancy?

**Comment by trallard** [comment]
What is the magenta colour being used for? 🤔 is this _absolutely_ necessary for the theme?
